.gitignore

```diff
@@ -6,6 +6,8 @@ Thumbs.db
 Externals/mGBA/version.c
 Source/Core/Common/scmrev.h
 # Ignore files output by build
+/cmake-build-debug
+/cmake-build-release
```

Comment:
Where exactly did you observe these paths? I've been building with CMake on Windows for a long time now and for me it just uses the `Build` directory that already has an ignore entry.

Reply:
CLion defaults to this path when building with its own configuration. It works the same as executing CMake + Make in the terminal, so there's no real reason not to add this.